.mergify.yml

```diff
@@ -0,0 +1,25 @@
+pull_request_rules:
+  - name: backport patches to 7.x branch
+    conditions:
+      - base=master
+      - label=backport-to-7.x
```

Comment:
We should use the existing labels like `v7.13.0` so people don't have to duplicate them.

Reply:
Yes, I added a comment to remember we need to set the labels accordingly.

For the record, we have a WiP automation to apply the same labels across all the observability repositories, therefore it will be quite simple to apply a new label everywhere.

Reply:
> I think we should use the existing labels like v7.13.0 so people don't have to duplicate them.

We backport too often after FF, sometimes right before release time leaving us with fixes that didn't make it into the release. From past experience we've been notoriously bad at telling which exact a fix/features is in. I think I would prefer to tell the target branch instead of the exact version.